.github/workflows/chaos.yml

```diff
@@ -27,99 +27,35 @@ jobs:
-          install_options: '--tarball-url-prefix https://nixos-nix-install-tests.cachix.org/serve'
-          extra_nix_config: |
-            experimental-features = nix-command flakes
-            access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-access-key-id: ${{ secrets.QA_AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.QA_AWS_SECRET_KEY }}
-          aws-region: ${{ secrets.QA_AWS_REGION }}
-          role-to-assume: ${{ secrets.QA_AWS_ROLE_TO_ASSUME }}
-          role-duration-seconds: 3600
-      - name: Set Kubernetes Context
-        uses: azure/k8s-set-context@v1
-        with:
-          method: kubeconfig
-          kubeconfig: ${{ secrets.QA_KUBECONFIG }}
-      - name: Cache Vendor Packages
-        uses: actions/cache@v2
-        id: cache-packages
-        with:
-          path: |
-            ~/.cache/go-build
-            ~/go/pkg/mod
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-
-      - name: Download Go Vendor Packages
-        if: steps.cache-packages.outputs.cache-hit != 'true'
-        run: nix develop -c make download
-      - name: Install Ginkgo CLI
-        run: nix develop -c make install
-      - name: Download Artifacts
-        uses: actions/download-artifact@master
-        with:
-          name: artifacts
-          path: artifacts
+      - uses: actions/checkout@f25a3a9f25bd5f4c5d77189cab02ff357b5aedeb # v2.4.1
       - name: Run Tests
-        env:
-          SELECTED_NETWORKS: localterra
-          NETWORK_SETTINGS: /home/runner/work/chainlink-terra/chainlink-terra/tests/e2e/networks.yaml
-          CHAINLINK_IMAGE: 795953128386.dkr.ecr.us-west-2.amazonaws.com/chainlink
-          CHAINLINK_VERSION: custom.${{ github.sha }}
-        run: |
-          PATH=$PATH:$(go env GOPATH)/bin
-          export PATH
-          cp -r artifacts packages-ts/gauntlet-terra-contracts/artifacts/bin
-          nix develop -c make test_chaos
-      - name: Publish Test Results
-        uses: mikepenz/action-junit-report@v2
-        if: always()
-        with:
-          report_paths: "./tests-chaos-report.xml"
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          check_name: Chaos Test Results
-      - name: Publish Artifacts
-        if: failure()
-        uses: actions/upload-artifact@v2.2.4
-        with:
-          name: test-logs
-          path: /home/runner/work/chainlink-terra/chainlink-terra/tests/e2e/logs
+        uses: smartcontractkit/ctf-ci-e2e-action@v1.0.0
+        with:
+          test_command_to_run: make test_chaos
+          cl_repo: 795953128386.dkr.ecr.us-west-2.amazonaws.com/chainlink
+          cl_image_tag: custom.${{ github.sha }}
+          download_artifacts_path: artifacts
```

Comment:
Consider renaming this property to be more clear those are contract artifacts. There are also other types of artifacts we could test, like for example the Gauntlet binary.

Reply:
That makes sense, I will update the ctf action